### PR TITLE
Append IDs like [#___] to Attendees Report

### PR DESCRIPTION
**Pull request by cliffordp**
Event name, each Ticket name, and each Attendee

- 8min before/after demo: https://share.getcloudapp.com/8Luj12OA
- 1min supplemental before/after demo for duplicate Attendee ID removed from Status column for RSVPs: https://share.getcloudapp.com/d5uvnB86

### CODE REVIEW

File: src/Tribe/Attendees_Table.php [hunk 1]
```diff
@@ -273,31 +273,41 @@ public function get_order_id_url( array $item ) {
...
 			$attendee_id .= ' &ndash; ';
 		}
 
 		?>
-			<div class="event-tickets-ticket-name">
-				<?php echo $attendee_id; ?>
-				<?php echo esc_html( $item['ticket'] ); ?>
-			</div>
+		<div class="event-tickets-ticket-name">
+			<?php echo $attendee_id; ?>
```
Comment (tr1b0t) [inline src/Tribe/Attendees_Table.php:305]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$attendee_id'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$attendee_id'.

<sub>hash: 829adde0191d6dd0a4305e5e948328c8</sub>

File: src/Tribe/Attendees_Table.php [hunk 2]
```diff
@@ -273,31 +273,41 @@ public function get_order_id_url( array $item ) {
...
+			<?php echo esc_html( $item['ticket'] ); ?>
+		</div>
 
-			<?php echo $this->get_row_actions( $item ); ?>
 		<?php
+		echo $this->get_row_actions( $item );
```
Comment (tr1b0t) [inline src/Tribe/Attendees_Table.php:310]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$this'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.

<sub>hash: 9f0ddea30cd9daaada673d266856ad81</sub>

File: src/admin-views/attendees.php [hunk 3]
```diff
@@ -11,19 +11,31 @@
...
 
 <div class="wrap tribe-report-page">
 	<?php if ( $show_title ) : ?>
-		<h1><?php esc_html_e( 'Attendees', 'event-tickets' ); ?></h1>
+		<h1>
+			<?php
+			echo esc_html(
+			// Translators: 1: the post title, 2: the post ID.
+				sprintf(
+					_x( 'Attendees for: %1$s [#%2$d]', 'event-tickets' ),
```
Comment (tr1b0t) [inline src/admin-views/attendees.php:32]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 8612ad0f6e9758fa6b2be0f09929c26c</sub>

File: src/admin-views/attendees.php [hunk 4]
```diff
@@ -90,16 +102,17 @@
...
-							/** @var Tribe__Tickets__Ticket_Object $ticket */
+							$ticket_name = sprintf( '%s [#%d]', $ticket->name, $ticket->ID );
 							?>
-						<li>
-							<strong><?php echo esc_html( $ticket->name ) ?>:&nbsp;</strong><?php
-							echo esc_html( tribe_tickets_get_ticket_stock_message( $ticket ) );
-						?></li>
-					<?php } ?>
+							<li>
+								<strong><?php echo esc_html( $ticket_name ) ?>:&nbsp;</strong><?php
```
Comment (tr1b0t) [inline src/admin-views/attendees.php:110]: * :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: a8df65c9aef50c6b363cfda6103eb10f</sub>

File: src/admin-views/attendees.php [hunk 5]
```diff
@@ -90,16 +102,17 @@
...
+								echo esc_html( tribe_tickets_get_ticket_stock_message( $ticket ) );
+								?></li>
```
Comment (tr1b0t) [inline src/admin-views/attendees.php:112]: :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 91db2997e16727019f37b4a71b12cafa</sub>

File: src/Tribe/Attendees_Table.php (hunk 1 above)
Comment (cliffordp) [reply src/Tribe/Attendees_Table.php:305]: purposefully not escaped to allow for ndash

File: src/Tribe/Attendees_Table.php [hunk 6]
```diff
@@ -273,31 +270,41 @@ public function get_order_id_url( array $item ) {
...
 			$attendee_id .= ' &ndash; ';
 		}
 
 		?>
-			<div class="event-tickets-ticket-name">
-				<?php echo $attendee_id; ?>
-				<?php echo esc_html( $item['ticket'] ); ?>
-			</div>
+		<div class="event-tickets-ticket-name">
+			<?php echo $attendee_id; ?>
```
Comment (tr1b0t) [inline src/Tribe/Attendees_Table.php:302]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$attendee_id'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$attendee_id'.

<sub>hash: 4bb9ea86ab4fcea27b57ac1e60847fb3</sub>

File: src/Tribe/Attendees_Table.php [hunk 7]
```diff
@@ -273,31 +270,41 @@ public function get_order_id_url( array $item ) {
...
+			<?php echo esc_html( $item['ticket'] ); ?>
+		</div>
 
-			<?php echo $this->get_row_actions( $item ); ?>
 		<?php
+		echo $this->get_row_actions( $item );
```
Comment (tr1b0t) [inline src/Tribe/Attendees_Table.php:307]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$this'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.

<sub>hash: 2dd83ca682fc8092e8beea3b3fdeeb71</sub>

**Pull request merged**